tests/drink/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "test-project-drink"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+drink = "0.8.6"
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = [
+    "derive",
+] }
+
+ink-wrapper-types = { path = "../../ink-wrapper-types", default-features = false, features = [
+    "drink",
+] }
+frame-system = "23.0.0"
+ink_metadata = "4.3.0"
+ink_primitives = "4.2.1"
```

Comment:
is this difference ok?

Reply:
I've changed all ink_* deps to `4.3.0` now. It seems to work with drink-only.